twisted/conch/insults/helper.py

```diff
@@ -177,7 +179,7 @@ def insertAtCursor(self, b):
 
     def _emptyLine(self, width):
         return [(self.void, self._currentFormattingState())
-                for i in xrange(width)]
```

Comment:
Wouldn't it be better to keep using xrange? On Python 2.7 you will create a list, and have the performance cost of doing that. Which you do not get if you keep using xrange from compat.